sdk/lib/collection/collection.dart

```diff
@@ -4,6 +4,10 @@
 
 /**
  * Classes and utilities that supplement the collection support in dart:core.
+ * 
+ * To use this library in your code:
+ *
+ *     import 'dart:collection';
```

Comment:
Why?
All of the core libraries use indentation. The only exception seems to be `uri.dart`.

Tbh I don't mind either, but I wouldn't ask patches to the core libraries to use one or the other.